shipitscript/src/shipitscript/pushlog_scan.py

```diff
@@ -33,7 +33,9 @@ def get_shippable_revision_build(branch, last_shipped_rev, cron_rev):
     # TODO use redo here to automatically retry
     resp = requests.get(URL.format(branch=branch), params={"full": "1", "fromchange": last_shipped_rev, "tochange": cron_rev})
```

Comment:
Note: including `branch=default` or something in the params doesn't work, I've already tried in:

https://hg.mozilla.org/releases/mozilla-esr68/json-pushes?full=1&branch=default&fromchange=ba35b4e926ed8da2b7796272948d9d86cfec8f8d&tochange=615d4ace089d57ddf11facdb678188fb41c0981b